metabase/metabase
# Navbar Improvements: Embedding

### Description

This changes prevents the logo button from being pressed when user has configured their embed with the `top_nav` option to `true` and the `side_nav` to `false`. This will prevent users from getting stuck on the home page with no way back.

This is the last change for the work to complete https://github.com/metabase/metabase/issues/36370

### How to verify

- create interactive embed with the `top_nav=true&side_nav=false` query params, make sure the page set is any page but the home page
- try to click the logo (nothing should happen)

### Demo

<img width="1165" alt="Screenshot 2024-01-23 at 11 29 05 AM" src="https://github.com/metabase/metabase/assets/7104357/b1417a20-372f-4689-a7d9-df3cd31397e7">
(you can see the link is disabled)

### Checklist
